src/components/BoxApproval/BoxApproval.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import { Stack, Button } from '@chakra-ui/react';
+import Box from './Box';
+import './BoxApproval.css';
+
+function BoxApproval() {
+  const [currentTab, setCurrentTab] = useState('requiresApproval');
+  // eslint-disable-next-line no-unused-vars
+  const [requiresApprovalBoxes, setRequiresApprovalBoxes] = useState([Box(), Box(), Box()]);
+  // eslint-disable-next-line no-unused-vars
+  const [approvedDeniedBoxes, setApprovedDeniedBoxes] = useState([Box(), Box()]);
+
+  return (
+    <div className="boxApproval">
+      <h1 className="boxApprovalHeading">Box Approval</h1>
+      <div className="boxApprovalMenu">
+        <Stack spacing={0} direction="row" align="center" className="buttonStack">
+          <Button
```

Comment:
In general, we want to separate our css styling and html/react components so let's move the styling on the buttons to the css file. Please do the same for Box.jsx.